ext/nnstreamer/tensor_converter/tensor_converter_python3.cc

```diff
@@ -365,18 +365,24 @@ extern "C" {
 void
 init_converter_py (void)
 {
-  registerExternalConverter (&Python);
   /** Python should be initialized and finalized only once */
-  Py_Initialize ();
+  if (!Py_IsInitialized()) {
+    Py_Initialize ();
+  }
+  registerExternalConverter (&Python);
 }
 
 /** @brief Destruct this object for tensor converter sub-plugin */
 void
 fini_converter_py (void)
 {
-  /** Python should be initialized and finalized only once */
-  Py_Finalize ();
   unregisterExternalConverter (Python.name);
+  /** Python should be initialized and finalized only once */
+  if (Py_IsInitialized()) {
+    /** Wait external converter is unregistered */
+    g_usleep (100000);
```

Comment:
Q. Really necessary? `unregisterExternalConverter` is asynchronous method?

Reply:
The python C-API seems operating asynchronously.
Without sleep time, a segmentation fault occurs after the test passes successfully.
This looks like an old problem with numpy and python C-API.

Reply:
Then, please comments, marking `TODO`

Reply:
Updated.